louie_pizza/menus/fixtures/menu_items.json

```diff
@@ -0,0 +1,262 @@
+[
```

Comment:
Moved these into fixtures so we're not reliant on a particular version of the SQLite DB in SCM. We could do away with storing the SQLite DB in `git` now and could just have users run `./manage.py migrate && ./manage.py loaddata menus/fixtures/menu_items.json` when they pull the project.